NullPointerException when starting the Product SP with v2.6.1

**Description:**
NullPointerException when starting the Product SP with v2.6.1

```
ERROR {msf4j-swagger} - [SCR] Error while trying to bind reference Reference[name = microserviceregsitry, interface = org.wso2.msf4j.MicroservicesRegistry, policy = dynamic, cardinality = 1..n, target = null, bind = addRegistry, unbind = removeRegistry]
	Details:
	Problematic reference = Reference[name = microserviceregsitry, interface = org.wso2.msf4j.MicroservicesRegistry, policy = dynamic, cardinality = 1..n, target = null, bind = addRegistry, unbind = removeRegistry]
	of service component = SwaggerDefinitionS
............
at org.eclipse.equinox.internal.util.impl.tpt.threadpool.Executor.run(Executor.java:70)
Caused by: java.lang.NullPointerException
	at org.wso2.msf4j.swagger.internal.SwaggerDefinitionSC.addRegistry(SwaggerDefinitionSC.java:48)
	... 106 more

```

**Suggested Labels:**
Bug

**Suggested Assignees:**
@Yasara123

**Affected Product Version:**
2.6.1

--- a/swagger/src/main/java/org/wso2/msf4j/swagger/internal/SwaggerDefinitionSC.java
+++ b/swagger/src/main/java/org/wso2/msf4j/swagger/internal/SwaggerDefinitionSC.java
@@ -29,6 +29,8 @@ import java.util.Dictionary;
 import java.util.Hashtable;
 import java.util.Map;
 
+import static org.wso2.msf4j.internal.MSF4JConstants.CHANNEL_ID;
+
 /**
  * OSGi service component for SwaggerDefinitionService.
  */
@@ -36,8 +38,7 @@ import java.util.Map;
         service = SwaggerDefinitionSC.class,
         immediate = true)
 public class SwaggerDefinitionSC {
-    private static final String CHANNEL_ID = "LISTENER_INTERFACE_ID";
-
+    
     @Reference(
             name = "microserviceregsitry",
             service = MicroservicesRegistry.class,

